Vagrantfile

```diff
@@ -86,6 +86,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   config.vm.define "database" do |database|
+    # SSH issues bringing up DB VM with base box 22.04
+    # Explicitly define downgraded version for use by database until fix 
+    database.vm.box = "bento/ubuntu-20.04"
```

Comment:
When configuring all machines with a "bento/ubuntu-22.04" base box I ran into issues successfully provisioning and bringing up the database VM. As part of booting the VM it would hang on `database: SSH auth method: private key` before failing with a timeout error. This has been [a reported issue on Vagrant before](https://github.com/hashicorp/vagrant/issues/11777), but all the previous solutions recommended in comments don't work. I previously ran into this issue once with Cicero and I was able to solve it by explicitly defining the `ssh auth.username` and `ssh.auth.password` in the Vagrantfile, but that did not work here either.

I settled on configuring a less-upgraded base box that doesn't have ssh issues (20.04) for the database and then configuring the more-upgraded (22.04) base box version for otp and app. I'm not sure if we're going to run into trouble with these two being different, but I struggled to manually install Python 3.10 for use in the Ubuntu 20.04 version so this was the only way I could figure out how to bring Python up past 3.8 in the app while also avoiding VirtualBox issues.